src/lib/user/methodsTypes.d.ts

```diff
@@ -0,0 +1,8 @@
+/** Types for the Friend's page */
```

Comment:
is `methodsTypes.d.ts` really your best bet? Maybe put it in `types` folder and call it `user.d.ts` with all the other user-related stuff.

Reply:
Yeah I did think of this and made it before reading this, stupid selection of names for the file.